src/app/order/list/list.component.spec.ts

```diff
@@ -1,20 +1,23 @@
-import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { HttpClientModule } from '@angular/common/http';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { RouterModule } from '@angular/router';
 
-import { ListComponent } from './list.component';
+import { OrderListComponent } from './list.component';
 
 describe('ListComponent', () => {
-  let component: ListComponent;
-  let fixture: ComponentFixture<ListComponent>;
+  let component: OrderListComponent;
+  let fixture: ComponentFixture<OrderListComponent>;
 
-  beforeEach(async(() => {
+  beforeEach(() => {
     TestBed.configureTestingModule({
-      declarations: [ ListComponent ]
+      imports: [ HttpClientModule, RouterModule ],
```

Comment:
should import `HttpClientTestingModule` instead

Reply:
👍  Fixed